app/policies/standup_meeting_policy.rb

```diff
@@ -1,8 +1,6 @@
 class StandupMeetingPolicy < ApplicationPolicy
-  alias_method :standup_meeting, :record
-
   def index?
-    user.admin? || matching_user?
+    group_member? || user.admin?
```

Comment:
fine with this for now but this is a strange one cause of the mixing/matching of what the "record" actually is. Not sure of a good solution to make this clearer.